clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -10414,6 +10414,9 @@ static void ProcessDeclAttribute(Sema &S, Scope *scope, Decl *D,
   case ParsedAttr::AT_SYCLSpecialClass:
     handleSimpleAttribute<SYCLSpecialClassAttr>(S, D, AL);
     break;
+  case ParsedAttr::AT_SYCLDeviceGlobal:
+    handleSimpleAttribute<SYCLDeviceGlobalAttr>(S, D, AL);
+    break;
```

Comment:
You can remove this codes. Add let SimpleHandler = 1; inside Attr.td file

Reply:
I think more changes will come from @schittir for this attribute, so I'm not sure if it will stay as a simple handler. @schittir , WDYT?

Reply:
Sorry i did not know about more changes coming from @schittir. Based on your patch, i thought about simple handler.
You can ignore this one then

Reply:
Yes, please leave it as it is. I will update it.

Reply:
Thanks for checking.

Reply:
Yeah, after looking at @schittir's changes, I think a simple handler inside Attr.td file makes more sense.